src/containers/explore/EntityDataContainer.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * @flow
+ */
+
+import React, { useEffect } from 'react';
+
+import styled from 'styled-components';
+import { List, Map } from 'immutable';
+import {
+  AppContentWrapper,
+  Spinner,
+  Typography
+} from 'lattice-ui-kit';
+import {
+  DataUtils,
+  ReduxUtils,
+  ValidationUtils,
+  useRequestState
+} from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+import type { FQN, Organization, UUID } from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import EntityNeighborsContainer from './EntityNeighborsContainer';
+import {
+  EXPLORE_ENTITY_DATA,
+  exploreEntityData,
+  exploreEntityNeighbors
+} from './actions';
+
+import {
+  CrumbItem,
+  CrumbLink,
+  Crumbs,
+  LinkButton
+} from '../../components';
+
+import { FQNS } from '../../core/edm/constants';
+import {
+  ENTITY_NEIGHBORS_MAP,
+  EXPLORE,
+  SELECTED_ENTITY_DATA
+} from '../../core/redux/constants';
+import { selectOrgDataSet, selectOrganization } from '../../core/redux/selectors';
+import { Routes } from '../../core/router';
+import { clipboardWriteText } from '../../utils';
+import { EntityDataGrid } from './components';
+
+const { getPropertyValue } = DataUtils;
+const { isPending } = ReduxUtils;
+const { isValidUUID } = ValidationUtils;
+
+const FlexContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 30px;
+`;
+
+type Props = {
+  dataSetDataRoute ?:string;
```

Comment:
```suggestion
  dataSetDataRoute :string;
```

Reply:
the crumbs are only displayed on the stand-alone page, so these are actually optional arguments. No need to pass them in the modal.